doc/source/whatsnew/v0.25.0.rst

```diff
@@ -551,7 +551,7 @@ with :attr:`numpy.nan` in the case of an empty :class:`DataFrame` (:issue:`26397
 ``__str__`` methods now call ``__repr__`` rather than vice versa
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-pandas has until now mostly defined string representations in a pandas objects'
+Pandas has until now mostly defined string representations in a pandas objects'
```

Comment:
```suggestion
pandas has until now mostly defined string representations in a pandas objects'
```

pandas is supposed to be lowercase (and throughout this file)

Reply:
Ok, I've corrected it